packages/app/src/debug/DebugResults.cy.tsx

```diff
@@ -0,0 +1,33 @@
+import DebugResults from './DebugResults.vue'
+import { defaultMessages } from '@cy/i18n'
+import { CloudRunStubs } from '@packages/graphql/test/stubCloudTypes'
+
+describe('<DebugResults />', () => {
+  it('mounts with default values', () => {
+    cy.mount(() => (<DebugResults/>))
+    cy.get('[data-cy=debug-results-holder]').then((items) => {
+      cy.get(`[title=${defaultMessages.runs.results.passed}]`).should('contain.text', 1)
+      cy.get(`[title=${defaultMessages.runs.results.failed}]`).should('contain.text', 7)
+      cy.get(`[title=${defaultMessages.runs.results.skipped}]`).should('contain.text', 6)
+      cy.get(`[title=${defaultMessages.runs.results.pending}]`).should('contain.text', 6)
+    })
+  })
+
+  it('shows the failed icon and the number of passed, skipped, pending, failed tests passed through gql props', () => {
```

Comment:
Definitely recommend writing with `mountFragment`, just do a search in the code base, there are many examples.